docs/guide/controller-programming.md

```diff
@@ -98,11 +243,49 @@ The call to the `wb_distance_sensor_get_value` function retrieves the latest val
 
 Note that some device return vector values instead of scalar values, for example these functions:
 
+%tab-component "language"
+%tab "C"
 ```c
 const double *wb_gps_get_values(WbDeviceTag tag);
 const double *wb_accelerometer_get_values(WbDeviceTag tag);
 const double *wb_gyro_get_values(WbDeviceTag tag);
 ```
+%tab-end
+
+%tab "C++"
+```cpp
+webots::GPS::const double *getValues() const;
+webots::Accelerometer::const double *getValues() const;
+webots::Gyro::const double *getValues() const;
+```
+%tab-end
+
+%tab "Python"
+```python
+GPS.getValues()
+Accelerometer.getValues()
+Gyro.getValues()
+
+#Returns the sensor measurement as an array of 3 floating point numbers: `[x, y, z]`.
+```
+%tab-end
+
+%tab "Java"
+```java
+public GPS::public double[] getValues();
+public Accelerometer::public double[] getValues();
+public Gyro::public double[] getValues();
```

Comment:
That looks horrible as well... What about this:
```suggestion
double[] GPS::getValues();
double[] Accelerometer::getValues();
double[] Gyro::getValues();
```

Reply:
This is why I asked for a suggestion in my very first comment...